examples/expandable-rows/index.js

```diff
@@ -84,7 +84,7 @@ class Example extends React.Component {
       expandableRowsOnClick: true,
       isRowExpandable: (dataIndex, expandedRows) => {
         // Prevent expand/collapse of any row after the 5th (but allow those already expanded to be collapsed)
-        if (expandedRows.data.length > 4 && expandedRows.data.filter(d => d.dataIndex === dataIndex).length === 0) return false;
+        if (dataIndex > 4 && expandedRows.data.filter(d => d.dataIndex === dataIndex).length === 0) return false;
```

Comment:
I think we can revert this, as it doesn't operate as intended, to disable any expandable row after the 4th (out of any rows). With your change, it will only disable according to the position of the row, which wasn't the intent.

Reply:
Hmm, I think the comment says otherwise? "Prevent expand/collapse of any row after the 5th". If we want to follow what the comment says, this change is kinda needed.

## Before my change (using expandedRows.data.length, the 5th row is still expandable (it shouldn't)

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/7077157/69017566-02b44a80-09fc-11ea-80f1-12c74177b5d2.gif)

## After my change

![ezgif com-video-to-gif(1)](https://user-images.githubusercontent.com/7077157/69017568-06e06800-09fc-11ea-8ae9-2968969bba4f.gif)

Reply:
Oh, and the "After my change" GIF section above was before the merging from 2 files into 1. This is the one after the merge. (see the non-expandable rows don't have toggle anymore)

![ezgif com-video-to-gif(2)](https://user-images.githubusercontent.com/7077157/69018400-6476b380-0a00-11ea-9e3f-ccf44313d7d6.gif)

Reply:
Haha, so what you pointed out to me made me realize that my comment was not clear. What I meant was, 5th as in total number of rows expanded. That was my intent. Feel free to change the comment to make that more clear!

Reply:
ohhhh right, sorry, I thought it means the "order" of the row instead of the "number of rows". All good now!